tests/end_to_end/typedef/context.c

```diff
@@ -0,0 +1,2 @@
+typedef int int_t;
+int_t func_00400090(int x, int_t *y);
```

Comment:
you need a -flags.txt file for this file to be used

(one thing I've considered is running all tests both with and without orig.c as context for some cheap extra coverage)

Reply:
Good catch - I figured this out late, and didn't `git add` it.